src/pages/Thanks/ThanksPage.vue

```diff
@@ -293,10 +294,9 @@ export default {
 		this.isGuest = this.receipt && !data?.my?.userAccount;
 
 		// Enable FTDs message from settings
-		const enableFtdMessageData = data?.general?.ftd_message_enable ?? null;
-		this.enableFtdMessage = enableFtdMessageData ? enableFtdMessageData.value : false;
+		this.isFtdMessageEnable = readBoolSetting(data, 'general.ftd_message_enable.value');
```

Comment:
Is this value now getting set properly to `false` in your local against dev?

Reply:
Yes, I'm getting `false` locally on this branch.
But also, getting `false` in main branch.